client/webserver/site/src/js/markets.ts

```diff
@@ -1097,7 +1097,7 @@ export default class MarketsPage extends BasePage {
     page.lotSize.textContent = Doc.formatCoinValue(mkt.cfg.lotsize, mkt.baseUnitInfo)
     page.rateStep.textContent = Doc.formatCoinValue(mkt.cfg.ratestep / rateConversionFactor)
 
-    if (!baseAsset?.wallet || !quoteAsset?.wallet) Doc.setVis(true, page.noWallet)
+    if (!baseAsset?.wallet && !quoteAsset?.wallet) Doc.setVis(true, page.noWallet)
```

Comment:
This is still broken. If both of the wallets are missing, it will not show the buttons to create the wallets. I think the `||` is fine, just remove the `else` on the next line. Also the no wallet message could be changed to say `Create a XXX wallet to trade` if only one of the wallets is missing.

Reply:
You’re correct, It used to show create btns for both wallets if they are missing, thanks for the review.